tests/unittests/operators/transient_jobs_tests.py

```
@@ -0,0 +1,126 @@
+
+from bson import ObjectId
+
+from fiftyone.factory.repos.execution_store import InMemoryExecutionStoreRepo
+from fiftyone.operators.store import (
+    ExecutionStore,
+    ExecutionStoreService,
+    TransientJobCoordinator,
+)
+
+
+def _coordinator(store=None):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_coordinator`

(ANN202)